pyglui/ui_elements.pxi

```diff
@@ -5,9 +5,9 @@ cdef class UI_element:
     '''
     The base class for all UI elements.
     '''
-    cdef readonly basestring label
     cdef readonly object uid
     cdef public FitBox outline
+    cdef basestring _label
```

Comment:
if you do this (which is fine). Please replace calls to self.label with self._label . The property call is python speed not c speed. The call to the field self._label is c speed.

I ll do this in a commit post this merge.